postgres/tests/conftest.py

```diff
@@ -16,7 +16,18 @@
 from .common import DB_NAME, HOST, PASSWORD, PORT, USER
 
 HERE = os.path.dirname(os.path.abspath(__file__))
-INSTANCE = {'host': HOST, 'port': PORT, 'username': USER, 'password': PASSWORD, 'dbname': DB_NAME, 'tags': ['foo:bar']}
+INSTANCE = {
+    'host': HOST,
+    'port': PORT,
+    'username': USER,
+    'password': PASSWORD,
+    'dbname': DB_NAME,
+    'tags': ['foo:bar'],
+    'query_settings': {
```

Comment:
This instance is shared for all tests, should this be done only on relevant tests? My assumption is that query settings are disabled by default.